raisinbread/RB_files/RB_files.sql

```diff
@@ -1,316 +1,316 @@
 SET FOREIGN_KEY_CHECKS=0;
 TRUNCATE TABLE `files`;
 LOCK TABLES `files` WRITE;
-INSERT INTO `files` (`FileID`, `SessionID`, `File`, `SeriesUID`, `EchoTime`, `CoordinateSpace`, `OutputType`, `AcquisitionProtocolID`, `FileType`, `InsertedByUserID`, `InsertTime`, `SourcePipeline`, `PipelineDate`, `SourceFileID`, `ProcessProtocolID`, `Caveat`, `TarchiveSource`, `ScannerID`, `AcqOrderPerModality`) VALUES (134,1053,'assembly/587630/V1/mri/native/demo_587630_V1_t1_001.mnc','1.3.12.2.1107.5.2.32.35008.2009052710234557661341584.0.0.0',0.00316,'native','native',44,'mnc','cecile',1558633727,NULL,NULL,NULL,NULL,0,27,1,1);
```

Comment:
I don't see why this file is modified here.. it doesn't look like the patch should affect it.

Reply:
I was confused about that as well. I used the `DB_dump_table_data.php` tool to extract the changes to raisinbread and for some reason this was included in the changes. I can remove this file from the PR just in case.

Reply:
@ridz1208 thoughts?

Reply:
the `HrrtArchiveID` column was removed at the same time as the changes were made

most likely removed in another PR and forgot to remove it from RB.

running the patch and exporting the dataset revealed the changes

Reply:
Hm ok. Should I try a fresh install of RB then and re-run the patch? Or would that not make a difference?

Reply:
@AlexandraLivadas that would not work since the column is now in the schema, it will always be automatically created when you resource RB. That column was added 22 days ago in #6142 

You could simply checkout the changes from this Pr. basically just ignore that file when submitting the changes. Another PR should be issue to apply the patch found in #6142 on RB

Reply:
Ok I can do that. I can also make the PR to apply the patch from #6142 today.

Reply:
@AlexandraLivadas that would be great.

one caveat to keep in mind however is the following. Because the PR creating the column was merged without the RB update going with it, you will find that the patch in the PR might fail when trying to create a column that already exists (because it was sourced from the schema). I have found that the easiest thing to do is to simply remove the `ADD column` command from the patch and any `ADD constraint` that might also fail for duplication before running the patch...

if it gets too complicated, let me know I can assist

Reply:
Ok thanks, I will let you know! For this PR, I can try to re-apply the patch with a fresh RB install and just ignore any files not related to this, but I should also try to get rid of the changes made to `RB_files.sql`, correct?

Reply:
depends you you are going at it. basically if you start from scratch, after running export_DB simply checkout any "unrelated" changes. if you wanna save this PR. simply reset to HEAD the files that are not supposed to be changed.

Reply:
Ok sounds good! Thank you

Reply:
Also, when you apply a new patch or a cleanup patch to RB you can enter the name of the patch in the [migration log](https://github.com/aces/Loris/blob/main/raisinbread/migration.md) this helps identify if the RB dataset has fallen behind. up until now I was mostly handling that part but any help is appreciated.

Reply:
Ok I will add that in this PR as well as the one I make for #6142